packages/grid-pro/test/edit-column-lit.test.js

```diff
@@ -1,3 +1,4 @@
+import './not-animated-styles.js';
```

Comment:
Since we have "select column" tests in this suite, I though it would be good to disable select animation.